### ISSUE TITLE
Baro Encoder: Switch for altimeter unit hPa/inHg

### ISSUE THREAD

**Issue by devpaw-de** [calibx/msfsdeck#221]
Maybe add a switch to change the altimeter unit. 
Could be on a short hold of the baro encoder.
I guess reading the current setting from the sim and using it for the indication on the deck would be an alternative.

**Comment by calibx** [comment]
I can't find a way to read configured preferences with SimConnect so I will add a config units panel for the plugin

**Comment by termigrator** [comment]
For me, it would be sufficient if the value in inches and hpa were visible at the loupedeck unit. There is still space to display both values.

**Comment by fjaroski** [comment]
Yeah, simconnect is limited in what is availible. I'am working in a A320 Fenix profile using spad.next and the software provide much more options, like adjusting pilot baro and co-pilot in the same dial in sync with is a requirement for Fenix. Switching units for baro is super easy. SimConnect is the problem. I want to find a away to edit Calibx profile but its in JS with I have little knowledge

**Comment by calibx** [comment]
I will fix this with the WASM module
